js/assign-non-enumerable/assign-non-enumerable.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @module {function} can-util/js/assign-non-enumerable/assign-non-enumerable assign-non-enumerable
+ * @parent can-util/js
+ * @signature `assignNonEnumerable(target, source)`
+ *
+ * A simplified version of [Object.assign](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign), which only accepts a single source argument
+ * and makes all assigned properties non enumerable and writable.
+ *
+ * ```js
+ * var assignNonEnumerable = require("can-util/js/assign-non-enumerable/assign-non-enumerable");
+ *
+ * var obj = {};
+ *
+ * assignNonEnumerable(obj, {
+ *   foo: "bar"
+ * });
+ *
+ * console.log(obj.foo); // -> "bar"
+ * ```
+ *
+ * @param {Object} target The destination object. This object's properties will be mutated based on the object provided as `source`.
+ * @param {Object} source The source object whose own properties will be applied to `target`.
+ *
+ * @return {Object} Returns the `target` argument.
+ */
+
+module.exports = function (d, s) {
+	for (var prop in s) {
+		Object.defineProperty(d, prop,{
+			enumerable: false,
+			writable: true,
+			value: s[prop]
```

Comment:
configurable false means the prop isn't deleted, right?  if this is meant to be like Object.assign but non-enumerable it should be configurable true:

<img width="1248" alt="screen shot 2017-05-11 at 1 35 55 pm" src="https://cloud.githubusercontent.com/assets/361671/25963271/b8f3f7d4-364e-11e7-829e-f345d5b6a32c.png">